earthpy/spatial.py

```diff
@@ -343,20 +354,20 @@ def bytescale(data, cmin=None, cmax=None, high=255, low=0):
 
     Parameters
     ----------
-    data : ndarray
+    data : numpy array
         image data array.
-    cmin : scalar, optional
+    cmin : int (optional)
         Bias scaling of small values. Default is ``data.min()``.
-    cmax : scalar, optional
+    cmax : int (optional)
         Bias scaling of large values. Default is ``data.max()``.
-    high : scalar, optional
-        Scale max value to `high`.  Default is 255.
-    low : scalar, optional
-        Scale min value to `low`.  Default is 0.
+    high : int (optional)
```

Comment:
Please keep default values in docstrings

Reply:
note that high and low are actually predefined so not optional. Let's move them to be the 2nd and third variables since they have predefined values.